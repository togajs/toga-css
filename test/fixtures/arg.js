/** @param {Type} [name] - Description. */
/** @param {Type} [name] Description. */
/** @param {Type} name - Description. */
/** @param {Type} name Description. */
/** @param {Type} [name] */
/** @param {Type} name */
/** @param [name] - Description. */
/** @param [name] Description. */
/** @param name - Description. */
/** @param name Description. */
/** @param [name] */
/** @param name */
