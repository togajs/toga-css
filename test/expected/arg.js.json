{
    "type": "Tunic",
    "blocks": [
        {
            "type": "Code",
            "contents": ""
        },
        {
            "type": "Comment",
            "description": "",
            "tags": [
                {
                    "tag": "param",
                    "type": "{Type}",
                    "name": "[name]",
                    "description": "Description."
                }
            ]
        },
        {
            "type": "Code",
            "contents": "\n"
        },
        {
            "type": "Comment",
            "description": "",
            "tags": [
                {
                    "tag": "param",
                    "type": "{Type}",
                    "name": "[name]",
                    "description": "Description."
                }
            ]
        },
        {
            "type": "Code",
            "contents": "\n"
        },
        {
            "type": "Comment",
            "description": "",
            "tags": [
                {
                    "tag": "param",
                    "type": "{Type}",
                    "name": "name",
                    "description": "Description."
                }
            ]
        },
        {
            "type": "Code",
            "contents": "\n"
        },
        {
            "type": "Comment",
            "description": "",
            "tags": [
                {
                    "tag": "param",
                    "type": "{Type}",
                    "name": "name",
                    "description": "Description."
                }
            ]
        },
        {
            "type": "Code",
            "contents": "\n"
        },
        {
            "type": "Comment",
            "description": "",
            "tags": [
                {
                    "tag": "param",
                    "type": "{Type}",
                    "name": "[name]"
                }
            ]
        },
        {
            "type": "Code",
            "contents": "\n"
        },
        {
            "type": "Comment",
            "description": "",
            "tags": [
                {
                    "tag": "param",
                    "type": "{Type}",
                    "name": "name"
                }
            ]
        },
        {
            "type": "Code",
            "contents": "\n"
        },
        {
            "type": "Comment",
            "description": "",
            "tags": [
                {
                    "tag": "param",
                    "name": "[name]",
                    "description": "Description."
                }
            ]
        },
        {
            "type": "Code",
            "contents": "\n"
        },
        {
            "type": "Comment",
            "description": "",
            "tags": [
                {
                    "tag": "param",
                    "name": "[name]",
                    "description": "Description."
                }
            ]
        },
        {
            "type": "Code",
            "contents": "\n"
        },
        {
            "type": "Comment",
            "description": "",
            "tags": [
                {
                    "tag": "param",
                    "name": "name",
                    "description": "Description."
                }
            ]
        },
        {
            "type": "Code",
            "contents": "\n"
        },
        {
            "type": "Comment",
            "description": "",
            "tags": [
                {
                    "tag": "param",
                    "name": "name",
                    "description": "Description."
                }
            ]
        },
        {
            "type": "Code",
            "contents": "\n"
        },
        {
            "type": "Comment",
            "description": "",
            "tags": [
                {
                    "tag": "param",
                    "name": "[name]"
                }
            ]
        },
        {
            "type": "Code",
            "contents": "\n"
        },
        {
            "type": "Comment",
            "description": "",
            "tags": [
                {
                    "tag": "param",
                    "name": "name"
                }
            ]
        },
        {
            "type": "Code",
            "contents": "\n"
        }
    ]
}
